### DISCUSSION TITLE
Is Renovate able to detect updates in a remote regitry different from the one set in the Dockerfile ?

### DISCUSSION THREAD

**Discussion by javaxiss** [renovatebot/renovate#23497] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.110.2

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hello,

I want to update my dockerfiles with new tags from a remote public registry. 

In a Dockerfile I can have the following as a base image: 
`FROM docker.myprivate-artifactory/new-images/team1/ubi/ubi8-minimal:8.7-1107`

Is there Renovate settings, to detect new ubi8-minimal updates from  another registry ex: registry.access.redhat.com/ubi8-minimal , or in the same private registry but from another path ex: docker.myprivate-artifactory/new-path/new-team/base-images/ubi8-minimal:8.7-1107? 

### What you tried so far.

I tried to use `matchPackagePrefixes` but I'm not sure it is the right way to do that. Renovate still fetching new dependencies in the private registry following the same image path.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by lmilbaum** [comment]
I think regex manager can work. Not sure though.

**Comment by viceice** [comment, marked as answer]
check registry aliases

**Reply by javaxiss** [reply]
Thank you ! Indeed registryAliases did the trick. However, I had to upgrade renovate to v36 on which a fix on registry aliases has been included :wink:

**Reply by marwin1991** [reply]
@Javaxiss How did you fix it?

**Reply by marwin1991** [reply]
@viceice maybe you can help? https://github.com/renovatebot/renovate/discussions/23573

**Reply by javaxiss** [reply]
As suggested by @viceice I configured a registryAliases entry in my project renovate settings (renovate.json) :
`"dockerfile": {
            "registryAliases": {
                "old.registry.local/old/path": "new.registry.local/new/images/path"
            }
    }
`

Renovate will look for new updates in new.registry.local/new/images/path instead of the old one that should be declared in your projects dockerfiles. The MR Renovate will create, are only  related to new tags not the whole path since we use aliases. That was what I needed.

**Comment by javaxiss** [comment]
> @viceice maybe you can help? #23573

in [#23573](https://github.com/renovatebot/renovate/discussions/23573), it would be a bug in Nexus or the Registry API. It seems like the same issue I'm having with Redhat public registry : [#23481](https://github.com/renovatebot/renovate/discussions/23481)
Renovate fixes the tags number by default to n=1000 but the API is ignoring that parameter and still returning random 50 tags ...
